### PR TITLE
KelvinVR: Cooling + Heating solution 

### PR DESCRIPTION
**Pull request by NaturalAutomatics**
All the methods for cooling and heating up solution created

### CODE REVIEW

File: MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+    Serial.println("KelvinVR Peltier Control Example");
+    Serial.println("================================");
+}
+
+void loop()
+{
+    // Cooling cycle
+    Serial.println("Starting cooling cycle...");
+    kelvin.startCooling(200); // Start cooling at about 78% intensity
```
Comment (NaturalAutomatics) [inline MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino:75]: Create additional method for Cooling cycle (Cooling down, blue LED ON, while cycle working fade effect running)

File: MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    kelvin.stopThermalControl();
+    for (int i = 0; i < idleDuration / 1000; i++)
+    {
+        Serial.print("Idle: ");
+        Serial.print(i + 1);
+        Serial.println(" seconds");
+        delay(1000);
+    }
+
+    // Heating cycle
```
Comment (NaturalAutomatics) [inline MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino:97]: For Heating cycle red LED fading effect, blue LED off (Include ARGB LED in new prototype for single module selection)

**Review by NaturalAutomatics — COMMENTED** [review]
Creating new method for Cool/Heat cycle and done

File: MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino (hunk 1 above)
Comment (AlexNoyanov) [reply MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino:75]: Done! This method is called void runThermalCycle(bool isCooling, int intensity, int durationSeconds) :
`    // Run a cooling cycle at 78% intensity for 30 seconds
    kelvin.runThermalCycle(true, 200, 30);`

File: MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino (hunk 2 above)
Comment (AlexNoyanov) [reply MetaTempDriver/Hardware/library/KelvinVR/examples/Heating-Cooling/Heating-Cooling.ino:97]: For now there are one two-coloured GR LED and additional single Blue LED. In next gen there will be one single RGB LED for all the colours

**Review by NaturalAutomatics — COMMENTED** [review]
Done! Created Thermal Cycles

**Pull request merged**